packages/Python/lldbsuite/test/lang/swift/dwarfimporter/TestSwiftDWARFImporter.py

```diff
@@ -0,0 +1,48 @@
+# TestSwiftDWARFImporter.py
```

Comment:
I'm leaning towards "let's remove all the copyrights" for tests, here. Also, the year is outdated :)